Ancestry.com US Federal Census.js

```diff
@@ -1,14 +1,15 @@
 {
-	"translatorID":"0dda3f89-15de-4479-987f-cc13f1ba7999",
-	"translatorType":4,
-	"label":"Ancestry.com US Federal Census",
-	"creator":"Elena Razlogova",
-	"target":"^https?://search.ancestry.com/(.*)usfedcen|1890orgcen|1910uscenindex",
-	"minVersion":"1.0.0b4.r1",
-	"maxVersion":"",
-	"priority":100,
-	"inRepository":true,
-	"lastUpdated":"2011-01-11 04:31:00"
+	"translatorID": "0dda3f89-15de-4479-987f-cc13f1ba7999",
+	"label": "Ancestry.com US Federal Census",
+	"creator": "Elena Razlogova",
+	"target": "^https?://search.ancestry.com/(.*)usfedcen|1890orgcen|1910uscenindex",
```

Comment:
this target regexp is wrong. Should be `"^https?://search.ancestry.com/.*(usfedcen|1890orgcen|1910uscenindex)"`